test/e2e/tests/ppom/ppom-blockaid-alert-simple-send.spec.js

```diff
@@ -32,13 +35,13 @@ const SEND_REQUEST_BASE_MOCK = {
 };
 
 async function mockInfura(mockServer) {
+  await mockMultiNetworkBalancePolling(mockServer);
   await mockServerJsonRpc(mockServer, [
     ['eth_blockNumber'],
     ['eth_call'],
     ['eth_estimateGas'],
     ['eth_feeHistory'],
     ['eth_gasPrice'],
-    ['eth_getBalance'],
```

Comment:
not needed, as we are mocking the balances using the mockMultiNetworkBalancePolling